src/helm/proxy/clients/auto_client.py

```diff
@@ -79,6 +79,7 @@ def get_client(self, request: Request) -> Client:
                 client = OpenAIClient(
                     api_key=self.credentials["openaiApiKey"],
                     cache_config=cache_config,
+                    tokenizer_client=self.get_tokenizer_client("huggingface"),
                     chat_gpt_client=chat_gpt_client,
```

Comment:
Do you mind removing `chat_gpt_client` and deleting `ChatGPTClient`? I don't think we need it now.

Reply:
OK, will do this in a new PR.

Reply:
Filed #1447 to track removing `ChatGPTClient`.